schemas/PollCreated.json

```diff
@@ -0,0 +1,19 @@
+{
+  "type": "object",
+  "description": "Triggered when a poll is created.",
+  "properties": {
+    "eventName": {
+      "enum": ["PollStarted"]
+    },
+    "isAnonymous": {
+      "description": "Whether this poll is anonymous.",
+      "type": "boolean"
+    },
+    "numberOfAnswers": {
+      "description": "Number of answers in the poll.",
+      "type": "integer"
```

Comment:
To me "option" would be a better naming. To me an "answer" is picking a specific option.

Reply:
In the matrix spec is called "answer". In the UI it is called "option". Not sure what nomenclature the analytics usually follow.

Reply:
Idk, staying aligned with the spec LGTM anyway 👍🏻